t/eth_test.t

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use Test::MockModule;
+use Test::Exception;
+use Test::TCP;
+
+use Future::AsyncAwait;
+use IO::Async::Loop;
+
+use Math::BigInt;
+use Math::BigFloat;
+use Net::Async::Blockchain::Transaction;
+use Net::Async::Blockchain::ETH;
+use Net::Async::Redis;
+
+no indirect;
+use Ryu::Async;
+
+my $mock_rpc = Test::MockModule->new("Net::Async::Blockchain::Client::RPC::ETH");
+my $mock_eth = Test::MockModule->new("Net::Async::Blockchain::ETH");
+
+my $loop = IO::Async::Loop->new();
+$loop->add(my $blockchain_eth = Net::Async::Blockchain::ETH->new());
+
+my $transaction_hash = '0x418aeca12f07c7010109b03d5d012f3c93f922ab520e831efb29075ae4903489';
+my $address          = '0x8295507db4b0d6a18f6c69be7d5484d5dac3ed9d';
+
+my $get_block_by_number = {
```

Comment:
Please change the name on this variable as well as `$get_transaction_receipt` and try referring to these variables instead of using the literal values from this mocked variable - I had to `Ctrl+F` and make sure the literal value [here](https://github.com/binary-com/perl-Net-Async-Blockchain/pull/40/files#diff-408508e97b188bafb63a2eaf5befb367R175) is matching the `timestamp` field in `$get_block_by_number`.

Same thing applies for [this line](https://github.com/binary-com/perl-Net-Async-Blockchain/pull/40/files#diff-408508e97b188bafb63a2eaf5befb367R144).

```suggestion
my $sample_block_by_number = {
```